packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -134,6 +134,7 @@ export const MediaComponent = defineComponent({
       controls: false,
       synchronize: true,
       autoplay: true,
+      uiOffset: new Vector3(0, 0, 0),
```

Comment:
why are we redeclaring this here, if the media component can be composed with the interactable component for the UI?

Reply:
I added this as an optional offset for the specialized placement of the mediacontrols xrui, which do not use the default interactable flow as of right now. happy to discuss further or make changes (this was a light touch to first fix mediacomponent, but it uses its own custom update for the xrui for logic to show or hide...as well as custom positioning) might try to converge this further